mw-webapp/src/component/accordion/Accordion.module.scss

```diff
@@ -0,0 +1,8 @@
+@import "src/styles/variables";
+
+.accordionRoot {
+    border-radius: $accordion-radius;
+    margin: 0 auto;
```

Comment:
why does accordeon has auto margin. What if I don't want to center it?

Reply:
Removed it